boards: st: nucleo_l152re: fix LED polarity in device tree

rdagher opened the pull request:
Closes #87424.

carlescufi commented:
Thanks for the PR! Please take a look at [our commit message guidelines](https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-message-guidelines) to find out how to format your commit messages, and at [our contribution workflow](https://docs.zephyrproject.org/latest/contribute/guidelines.html#contribution-workflow) to understand how to update your Pull Request.

rdagher committed:
boards: st: nucleo_l152re: fix LED polarity in device tree

Fix incorrectly defined LED polarity (GPIO_ACTIVE_LOW → GPIO_ACTIVE_HIGH)
to match hardware schematic and ensure consistent GPIO API behavior.

Fixes: #87424
Link: https://www.st.com/en/evaluation-tools/nucleo-l152re.html#cad-resources

Signed-off-by: Roudy Dagher <roudy.dagher@gmail.com>
